### PR TITLE
[Enhancement] Employee Address moved to Address

### PR DESCRIPTION
**Pull request by KanchanChauhan**
Merge this as well : https://github.com/frappe/frappe/pull/2956

screen shot 2017-04-06 at 11 58 38 am

### CODE REVIEW

File: erpnext/patches/v8_0/employee_address_update.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	move_employee_address_to_address()
+
+def move_employee_address_to_address():
+	employee_addresses = frappe.db.sql("""select name, employee_name, company, permanent_accommodation_type, current_accommodation_type,
+						current_address, permanent_address from tabEmployee 
+						where
```
Comment (nabinhait) [inline erpnext/patches/v8_0/employee_address_update.py]: don't use so many tabs, use only one extra tab

File: erpnext/patches/v8_0/employee_address_update.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		if employee_address.permanent_address:
+			permanent_address = get_formatted_address(employee_address.permanent_address)
+			create_new_address(employee_address.name, employee_address.employee_name, permanent_address, employee_address.company, "Permanent", employee_address.permanent_accommodation_type)
+		if employee_address.current_address:
+			current_address = get_formatted_address(employee_address.current_address)
+			create_new_address(employee_address.name, employee_address.employee_name, current_address, employee_address.company, "Current", employee_address.current_accommodation_type)
+
+def get_formatted_address(address):
+	formatted_address = frappe._dict()
+	address = address.split("\n")
```
Comment (nabinhait) [inline erpnext/patches/v8_0/employee_address_update.py]: lets call it address_lines

File: erpnext/patches/v8_0/employee_address_update.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+	new_address.address_type = address_type
+	new_address.address_line1 = address.address_line1
+	new_address.city = address.city
+	new_address.address_line2 = address.address_line2 or ''
+	new_address.pincode = address.pincode or ''
+	new_address.country = address.country or frappe.get_value('Company', company, 'country')
+	new_address.state = address.state or ''
+	new_address.county = address.county or ''
+	new_address.append('links', { "link_doctype": "Employee", "link_name": name })
+	new_address.insert()
```
Comment (nabinhait) [inline erpnext/patches/v8_0/employee_address_update.py]: Set flags for `ignore_validate` and `ignore_mandatory`

**Review by nabinhait — CHANGES_REQUESTED** [review]

**Comment by KanchanChauhan** [comment]
@nabinhait All done

**Comment by nabinhait** [comment]
there is a huge chance to map the data in wrong field and also a chance to loosing data. Closing the PR

**Pull request closed without merging**